pkg/apis/apiserver/v1alpha1/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// v1alpha1 API definitions for KubeCarrier
+package v1alpha1
```

Comment:
I would like to start with just `v1`, while the whole project is in alpha, also our APIs need to be considered alpha and I don't want to support multiple `alpha`, `beta`, etc api versions. If we need to do breaking API changes we will bump our major version.

In difference to the kube-apiserver, new apiversions here don't come with no cost overhead, so we should keep it simple.

Reply:
what's the cost overhead you speak off? the `v1alpha1` makes a pretty nice semantic how stable this API is, and it's something I really enjoy regarding the API versioning from kubernetes.